src/createServer.js

```diff
@@ -1,11 +1,233 @@
+/* eslint-disable function-paren-newline */
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const router = express.Router();
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+  const mockUsers = [];
+  const mockExpenses = [];
```

Comment:
mockUsers and mockExpenses could both be any type o hashmap instead of arrays.
meaning, both Object and Map classes could handle better requests for finding, deleting and so on